routes/api.php

```diff
@@ -37,7 +37,9 @@
         Route::post('subscribe', 'UserController@subscribe');
         Route::get('users/me', 'UserController@me');
         Route::get('users/notifications', 'UserController@listNotifications');
+        Route::get('users/notifications/readAllNotification', 'UserController@readAllNotification');
```

Comment:
The endpoint would be
users/notifications/read-all

It doesn't make sense to use camel case in the URL.